src/hotspot/share/runtime/mutexLocker.cpp

```diff
@@ -250,12 +250,12 @@ void mutex_init() {
   def(SharedDictionary_lock        , PaddedMutex  , leaf,        true,  _safepoint_check_always);
   def(ClassInitError_lock          , PaddedMonitor, leaf+1,      true,  _safepoint_check_always);
   def(Module_lock                  , PaddedMutex  , leaf+2,      false, _safepoint_check_always);
-  def(InlineCacheBuffer_lock       , PaddedMutex  , leaf,        true,  _safepoint_check_never);
+  def(InlineCacheBuffer_lock       , PaddedMutex  , nosafepoint-1, true,  _safepoint_check_never);
```

Comment:
Why -1 ?

Reply:
It depends on CompiledIC_lock
30_CompiledIC_lock:def(CompiledIC_lock               , PaddedMutex , nosafepoint, _safepoint_check_never, true);

Reply:
Okay